code/FulpstationCode/fulp_medical.dm

```diff
@@ -0,0 +1,346 @@
+/*
```

Comment:
What's up with this file?

Reply:
That contains all the medical items in it. The /* ends after the title with */

Reply:
Cool, those items are all listed in the description?